Stripe/STPCustomer+Private.h

```diff
@@ -0,0 +1,26 @@
+//
+//  STPCustomer+Private.h
+//  Stripe
+//
+//  Created by Ben Guo on 12/18/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import "STPCustomer.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface STPCustomer ()
+
+/**
+ Updates the customer's sources and defaultSource with a Customer API response.
```

Comment:
I wonder if it's worth changing this from "Updates" to "Replaces", since you're getting all new objects?

Reply:
👍